doc/source/upstream-bug-fixes.rst

```diff
@@ -4,645 +4,651 @@
  List of upstream |MySQL| bugs fixed in |Percona Server| 5.5
 =============================================================
 
++-------------------------------------------------------------------------------------------------------------+
+|:Upstream bug: :mysqlbug:`53588` - Blackhole : Specified key was too long; max key length is 1000 bytes      |
+|:JIRA bug: :psbug:`1126`                                                                                     |
+|:Upstream state: Verified (checked on 2018-01-30)                                                            |
+|:Fix Released: :rn:`5.5.59-38.11`                                                                            |
+|:Upstream fix: N/A                                                                                           |
 +-------------------------------------------------------------------------------------------------------------+
 |:Upstream bug: :mysqlbug:`83264` - uint3korr should stop reading four instead of three bytes on x86          |
-|:Launchpad bug: :bug:`1628417`                                                                               |
+|:JIRA bug: :psbug:`3567`                                                                                     |
```

Comment:
Please avoid extra whitespace at the end of the line (it seems that "JIRA bug" in this PR has it)

Reply:
The comment still stands

Reply:
Highlighted trailing whitespaces here are just a diff visual effect - the whole table is filled with spaces)